hadoop-ozone/interface-client/src/main/resources/proto.lock

```diff
@@ -83,6 +83,10 @@
                 "name": "DeleteKeys",
                 "integer": 38
               },
+              {
+                "name": "RenameKeys",
+                "integer": 39
+              },
```

Comment:
Please remove changes to `proto.lock` from the patch.  See HDDS-3991 for details.

Reply:
Thanks for @adoroszlai 's advice. Fixed the issues mentioned above. Can you help take another look?